geth-patches/README.md

```diff
@@ -59,25 +51,23 @@ Once patches applied, you might want to inspect changes between current vendored
 diff -Nru -x "*_test.go" -x "vendor" -x ".git" -x "tests" -x "build" --brief $GOPATH/src/github.com/status-im/go-ethereum $GOPATH/src/github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum
 ```
 
-# Vendor patched version
-## Using `dep` tool
-
-TBD
+4. Push `develop` branch to our remote, rewriting history
+```
+git push -f origin develop
+```
 
-## Manually
-This method should be used only while `dep` tool workflow is not set up.
+#### II. In `status-go` repository
 
+1. Update vendored `go-ethereum` (note that we use upstream's address there, the link to our fork is in `Gopkg.toml`)
```

Comment:
`[...] the link to our fork is in [...]` maybe `[...] we override the download link to our fork address in [...]`